### DISCUSSION TITLE
ohmyzsh not working on my iSH App

### DISCUSSION THREAD

**Discussion by unknownbrian008** [ohmyzsh/ohmyzsh#11988] [Help] [answered]
I installed ohmyzsh on my iSH App but still doesn't work despite changing the theme![image](https://github.com/ohmyzsh/ohmyzsh/assets/62049901/a6d65baf-6595-4527-a791-f36663a5ba5a)

**Comment by carlosala** [comment, marked as answer]
Try to follow this thread https://github.com/ish-app/ish/issues/1361
